server/src/routes/article.route.js

```diff
@@ -13,17 +20,15 @@ const {
   updateArticleSchema,
   deleteArticleSchema
 } = validateSchema;
-const {
-  createArticle,
-  getSingleArticle,
-  getAllArticles,
-  updateArticle,
-  deleteArticle
-} = articleController;
+
+// articles
+
+// CRUD
 
 router.post('/', checkToken, createArticleSchema, validateResult, asyncErrorHandler(createArticle));
+router.get('/', asyncErrorHandler(getAllArticles));
+router.get('/', checkToken, asyncErrorHandler(getAllArticlesByStatus));
```

Comment:
These two endpoints are the same `get /` don't you think this means we should the same controller.